travel_platform/izi_travel_client.py

```diff
@@ -0,0 +1,412 @@
+            content_provider_uuid = content_provider.get("uuid", "")
+            content = museum_data.get("content", []).pop()
+            audio = content.get("audio", [])
+            audio_uuid = audio[0].get("uuid", "") if audio else ""
+            detail_museum_info.update({
+                "name": content_provider.get("name", ""),
+                "audio": self._prepare_audio_url(content_provider_uuid,
+                                                 audio_uuid),
+                "description": content.get("desc", ""),
+                "address": museum_data.get("contacts", {}),
+                "reviews": museum_data.get("reviews", {}),
+                "images": []
+
+            })
+            images = content.get("images", [])
+
+            for image in images:
+                im_url = self._prepare_image_url(content_provider_uuid,
+                                                 image.get("uuid", ""))
+                detail_museum_info["images"].append(im_url)
+            return detail_museum_info
+        return detail_museum_info
+
+    def get_tourist_attractions(self, tour_uuid, params=None):
+        """
+        Get list of tourist attractions included in a tour.
+        Example response:
+        [
+        {"status": .., "category": .., "placement": "..",
+        "uuid": "...", "distance": "..", "country": {...},
+        "languages": [..], "reviews": {..}, "map": {...},
+        "duration": ..., "content": [
+            {"language": "..", "title": "...", "playback": "..."},
+            "images": [{..}, {..}], "children": [
+            {...}, {...}, {...}]
+            ]
+        ]
+        Args:
+            tour_uuid (str): Tour uuid.
+            params (dict): Any additional url parameters.
+        Returns:
+            list: Tourist attractions.
+        """
+        url = self._prepare_base_url(self.TOUR_ENDPOINT.format(
+            tour_uuid=tour_uuid))
+
+        if params is None:
+            prepared_params = {"includes": "all,city,country",
+                               "except": "translations,publisher,download"}
+            params = self._prepare_params(**prepared_params)
+        response = self._make_request(url, **params)
+        tourist_attractions = []
+
+        if response:
+
+            tour_data = response.pop().get("content", []).pop()
+            for child in tour_data.get("children", []):
+
+                if child.get("type") == "tourist_attraction":
+                    attraction = {
+                        "title": child.get("title", ""),
+                        "description": child.get("desc", ""),
+                        "location": child.get("location", {}),
+                        "language": child.get("language", ""),
+                        "images": child.get("images", []),
+                        "attr_uuid": child.get("uuid", "")
+                    }
+                    tourist_attractions.append(attraction)
+            return tourist_attractions
```

Comment:
What is the reason of this return if we also have same return after this `if` condition?